src/commands/addBinding/BindingNameStep.ts

```diff
@@ -0,0 +1,56 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import { FunctionConfig } from '../../FunctionConfig';
+import { localize } from '../../localize';
+import { StringPromptStep } from '../createFunction/genericSteps/StringPromptStep';
+import { IBindingWizardContext } from './IBindingWizardContext';
+
+export class BindingNameStep extends StringPromptStep {
+    private _functionConfig: FunctionConfig | undefined;
+
+    public async getDefaultValue(wizardContext: IBindingWizardContext): Promise<string | undefined> {
+        const defaultValue: string | undefined = await super.getDefaultValue(wizardContext);
+        if (defaultValue) {
+            let uniqueValue: string = defaultValue;
+
+            let count: number = 1;
+            while (count < 100) {
```

Comment:
I feel like 100 should be an optional parameter or something.  100 seems pretty arbitrary-- granted I'm not a Functions developer so I dunno if anyone would ever hit the limit 😋

Reply:
Oh it's super arbitrary. I'll make it a `const` and name it `maxTries`, but I'm not gonna do an optional parameter because no one is every gonna pass in a different value. The goal was just to pick a number that is so large it'll probably never be hit for this scenario but so that there's no possibility of an infinite loop if there's some sort of bug in `bindingExists`